plugins/dependencies/src/main/kotlin/dependencies/JakeWharton.kt

```diff
@@ -72,4 +73,88 @@ object JakeWharton : IsNotADependency {
          */
         val shimo = module("shimo")
     }
+
+    /**
+     * RxJava binding APIs for Android's UI widgets. This dependency group is for RxJava2.
+     *
+     * GitHub page: [JakeWharton/RxBinding](https://github.com/JakeWharton/RxBinding/)
+     */
+    val rxBinding3 = RxBinding3
+
+    /**
```

Comment:
@mgray88 Kdoc isn't required here, I think. @LouisCAD can you confirm?

Reply:
I can confirm.